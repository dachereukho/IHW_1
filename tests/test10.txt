9
0
1
26
78
-100
4
7
7
25