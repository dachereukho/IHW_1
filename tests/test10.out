1 78 4 7 0 26 -100 7 25